tamr_toolbox/low_latency/llm.py

```diff
@@ -156,7 +152,7 @@ def llm_query(
         response = match_client.post(url, json=json_recs)
 
         # Process responses
-        if response.ok:
+        if response.successful():
```

Comment:
`response.successful()` actually returns the response itself, after checking that the status is okay.  A slightly better way to use this is just modify the above line to be 
```python
        response = match_client.post(url, json=json_recs).successful()
```
If you want to handle an unsuccessful response you can wrap this in a `try-except` (this will raise an `HttpError`).